src/CarsDataTable.js

```diff
@@ -1,12 +1,23 @@
 import React from 'react';
+import Car from "./Car";
+import Titles from "./Titles";
+import "./CarsDataTable.css";
 
 const CarsDataTable = (props) => {
+    const allCars = props.cars.map((car, index) => (
+        <tr key={index}><Car carParams={car} /></tr>
+    ))
+const tableStyle = {border: "2px solid grey", textAlign: "center", padding: 0}
+const divStyle = {textAlign: "center"}
   return (
-    <div>
+    <div style={divStyle} ClassName="table">
```

Comment:
typo `className`

Also "table" is not the best name for the class, especially so the element that you use it on is a `<div>` not a `<table>`. The name should also be more specific like `"carsDataTableWrapper" so that it's easy to understand what that CSS class is for.